linker/shared/src/test/scala/org/scalajs/linker/testutils/TestIRBuilder.scala

```diff
@@ -24,6 +26,8 @@ object TestIRBuilder {
   implicit val noPosition: ir.Position = ir.Position.NoPosition
 
   val EAF = ApplyFlags.empty
+  val EMF = MemberFlags.empty
+  val EOH = OptimizerHints.empty
 
   val emptyOptHints: OptimizerHints = OptimizerHints.empty
```

Comment:
There are a bunch of places in the test suite where these can be used (n.b. `LinkerTest`). Also note that `EOH` and `emptyOptHints` are exactly the same.

I'd recommend making a separate commit where you introduce these, and replace ad-hoc usage everywhere. And in the next commit you actually fix the issue.